src/main/kotlin/com/cjbooms/fabrikt/generators/client/OkHttpClientGenerator.kt

```diff
@@ -10,8 +10,10 @@ class OkHttpClientGenerator(
     packages: Packages,
     api: SourceApi
 ) {
-    private val simpleClientGenerator = OkHttpSimpleClientGenerator(packages, api)
-    private val enhancedClientGenerator = OkHttpEnhancedClientGenerator(packages, api)
+    private val simpleClientGenerator =
+        OkHttpSimpleClientGenerator(packages, api)
+    private val enhancedClientGenerator =
+        OkHttpEnhancedClientGenerator(packages, api)
```

Comment:
is this ktlint? Looks like unnecessary line breaks

Reply:
I don't think so, probably some intellij auto-formatting mess up. 
will fix the format